constants/rainbowTheme.ts

```diff
@@ -0,0 +1,58 @@
+import { Theme } from "@rainbow-me/rainbowkit";
+
+const darkGrey = "#1A1B1F";
+
+// TODO move this into the design system library and use variables
```

Comment:
@adamsoffer here's the styling for rainbow - let me know what you think is the best place for this to live (and I'm unsure of how to use the variables in the design system lib in JS)